### PR TITLE
fixed setup file

### PR DESCRIPTION
**Pull request by norton120**
## Pull request checklist

Before submitting your PR, please review the following checklist:

- [x] Consider adding a unit test if your PR resolves an issue.
- [x] All new and existing tests pass locally (`palm test`)
- [x] Lint (`palm lint`) has passed locally and any fixes were made for failures
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Breaking changes

- [ ] Check if this pull request introduces a breaking change

## What does this implement/fix? Explain your changes.
If you try to scaffold a new plugin it will fail, looking for a setup template that is not named to match the actual template. 
This fixes that.

### CODE REVIEW

File: palm/cli.py [hunk 1]
```diff
@@ -90,18 +90,12 @@ def get_version():
...
+        ("docker --version", "Docker is not installed, please install it first",),
         (
             "docker-compose --version",
             "Docker Compose is not installed, please install it first",
         ),
-        (
-            "docker ps",
-            "Docker is not running, please start it first",
-        ),
+        ("docker ps", "Docker is not running, please start it first",),
```
Comment (jakeberesford-palmetto) [inline palm/cli.py:98]: I agree this looks better... but `black` does not. Linting ablaze with this change

**Review by jakeberesford-palmetto — CHANGES_REQUESTED** [review]
Thanks for fixing the issue, unfortunately your test doesn't pass

**Comment by norton120** [comment]
blech; I think I may have accidentally finished these fixes on my fork's develop. will update and push now

**Review by jakeberesford-palmetto — APPROVED** [review]
Thanks for fixing!

**Review by emekdahl-palmetto — APPROVED** [review]

**Pull request merged**